basic/src/test/java/org/jboss/hal/testsuite/test/configuration/tools/ELResolverTestCase.java

```diff
@@ -0,0 +1,180 @@
+
+        addProperty(NAME, VALUE);
+
+        clickExpressionResolver();
+        getWindowFragment().getEditor().text("input", "${" + NAME + ":default_value}");
+        getWindowFragment().clickButton("Resolve");
+
+        String output = getWindowFragment().getEditor().text("output");
+        getWindowFragment().clickButton("Done");
+        String expected = "server-one=" + VALUE + "\nserver-two=" + VALUE + "\n";
+        assertEquals(expected, output);
+
+        removeProperty();
+    }
+
+    @Test
+    public void resolveServerPropertyWithSameValue() throws Exception {
+
+
+        goToServerProperties("server-one");
+        addProperty(NAME, VALUE);
+
+        goToServerProperties("server-two");
+        addProperty(NAME, VALUE);
+
+        clickExpressionResolver();
+        getWindowFragment().getEditor().text("input", "${" + NAME + ":default_value}");
+        getWindowFragment().clickButton("Resolve");
+
+        String output = getWindowFragment().getEditor().text("output");
+        getWindowFragment().clickButton("Done");
+        String expected = "server-one=" + VALUE + "\nserver-two=" + VALUE + "\n";
+        assertEquals(expected, output);
+
+        goToServerProperties("server-one");
+        removeProperty();
+
+        goToServerProperties("server-two");
+        removeProperty();
+    }
+
+    @Test
+    public void resolveServerPropertyWithDifferentValue() throws Exception {
+        String reverseValue = new StringBuilder(VALUE).reverse().toString();
+
+        goToServerProperties("server-one");
+        addProperty(NAME, VALUE);
+
+        goToServerProperties("server-two");
+        addProperty(NAME, reverseValue);
+
+        clickExpressionResolver();
+        getWindowFragment().getEditor().text("input", "${" + NAME + ":default_value}");
+        getWindowFragment().clickButton("Resolve");
+
+        String output = getWindowFragment().getEditor().text("output");
+        getWindowFragment().clickButton("Done");
+        String expected = "server-one=" + VALUE + "\nserver-two=" + reverseValue + "\n";
+        assertEquals(expected, output);
+
+        goToServerProperties("server-one");
+        removeProperty();
+
+        goToServerProperties("server-two");
+        removeProperty();
+    }
+
+    @Test
+    public void resolveServerPropertyDefinedOnOneServer() throws Exception {
+        goToServerProperties("server-one");
+        addProperty(NAME, VALUE);
+
+        clickExpressionResolver();
+        getWindowFragment().getEditor().text("input", "${" + NAME + ":default_value}");
+        getWindowFragment().clickButton("Resolve");
+
+        String output = getWindowFragment().getEditor().text("output");
+        getWindowFragment().clickButton("Done");
+        String expected = "server-one=" + VALUE + "\nserver-two=default_value\n";
+        assertEquals(expected, output);
+
+        goToServerProperties("server-one");
+        removeProperty();
+    }
+
+
+
+    /* UTILS */
```

Comment:
I suggest to extract all UTILS methods to UI abstraction class .. something like `ServerConfigurationPage` to separate the UI and test logic.